content/posts/python/object-is-not-subscriptable.md

```diff
@@ -5,14 +5,9 @@ summary: "Fixing this cryptic Python TypeError"
 
 ## Overview {#overview}
 
-You get an error complaining:
-`TypeError: object is not subscriptable`
+This problem is caused by trying to access an object that cannot be indexed as though it can be accessed via an index.
 
-Specific examples:
-* `TypeError: 'type' object is not subscriptable`
-* `TypeError: 'function' object is not subscriptable`
```

Comment:
Oh... I see you've moved these to lower on the page. The one problem with that is that I believe that gives them a lot less weight in Google's indexing, but they are the main errors that would lead to this page being useful to people.